tests/data/comments8.py

```diff
@@ -0,0 +1,101 @@
+# The percent-percent comments are Spyder IDE cells.
+# Both `#%%`` and `# %%` are accepted, so `black` standardises
+# to the latter.
+
+#%%
+def func():
+    x = """
+    a really long string
+    """
+    lcomp3 = [
```

Comment:
Can we shorten this code? The test can focus on just `#%%`, so no need for everything else.

Reply:
sure, I was meaning it to be a `--preview` version of `comments3.py` - but shortening it and just testing `# %%` seems fine